src/arrayMethodSort.js

```diff
@@ -4,8 +4,34 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) => {
+    if (String(a) > String(b)) {
+      return 1;
+    } else if (String(a) < String(b)) {
+      return -1;
+    } else {
+      return 0;
+    }
+  }) {
```

Comment:
Difficult to understand and you can simplify this
```suggestion
  [].__proto__.sort2 = function(compareFunction = (a, b) => a.toString() > b.toString()) {
```